Packages/MIES/MIES_NeuroDataWithoutBorders.ipf

```diff
@@ -594,6 +597,27 @@ Function NWB_ExportAllData(nwbVersion, [overrideFilePath, writeStoredTestPulses,
 	LOG_AddEntry(PACKAGE_MIES, "end", keys = {"size [MiB]"}, values = {num2str(NWB_GetExportedFileSize())})
 End
 
+/// @brief Wait for ASYNC nwb writing to finish
+///
+/// Currently wait up to 10min (NWB_ASYNC_WRITING_TIMEOUT * NWB_ASYNC_MAX_ITERATIONS),
+/// everything above 5s is considered a bug.
+Function NWB_ASYNC_FinishWriting(string panelTitle)
+
+	string workload, msg
+	variable i
+
+	workload = NWB_ASYNC_WorkLoadName(panelTitle)
+
+	for(i = 0; i < NWB_ASYNC_MAX_ITERATIONS; i += 1)
+		if(!ASYNC_WaitForWLCToFinishAndRemove(workload, NWB_ASYNC_WRITING_TIMEOUT))
+			return NaN
+		endif
+
+		sprintf msg, "Waiting for NWB writing thread %d/%d.\r" i + 1, NWB_ASYNC_MAX_ITERATIONS
```

Comment:
@t-b I can't seem to change NWB_ASYNC_WRITING_TIMEOUT and NWB_ASYNC_MAX_ITERATIONS so that this error message is triggered.

e.g., 
```
static Constant NWB_ASYNC_WRITING_TIMEOUT = .0000000000001
static Constant NWB_ASYNC_MAX_ITERATIONS  = 1
```